packages/wonder-blocks-grid/components/row.test.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+
+import * as React from "react";
+
+import {
+    MEDIA_DEFAULT_SPEC,
+    MediaLayoutContext,
+} from "@khanacademy/wonder-blocks-layout";
+import Row from "./row.js";
+import Cell from "./cell.js";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+describe("Row", () => {
+    beforeEach(() => {
+        unmountAll();
+    });
+
+    describe("large", () => {
+        it("should render Cells with largeCols and cols", () => {
+            // Arrange
+            const wrapper = mount(
+                <div>
+                    <MediaLayoutContext.Provider
+                        value={{
+                            overrideSize: undefined,
+                            ssrSize: "large",
+                            mediaSpec: MEDIA_DEFAULT_SPEC,
+                        }}
+                    >
+                        <Row>
+                            <Cell cols={1}>cols</Cell>
+                            <Cell largeCols={1}>largeCols</Cell>
+                            <Cell mediumCols={1}>mediumCols</Cell>
+                            <Cell smallCols={1}>smallCols</Cell>
+                        </Row>
+                    </MediaLayoutContext.Provider>
+                </div>,
+            );
+
+            // Act
+            const text = wrapper.text();
+
+            // Assert
+            expect(text).toEqual("colslargeCols");
+        });
+
+        it("should throw if there are too many columns", async () => {
+            // Arrange
+
+            // Prevent jest complaining when JSDOM console.error's an exception
+            jest.spyOn(console, "error");
+            // eslint-disable-next-line no-console
+            console.error.mockImplementation(() => {});
+
+            const render = () => {
+                mount(
+                    <div>
+                        <MediaLayoutContext.Provider
+                            value={{
+                                overrideSize: undefined,
+                                ssrSize: "large",
+                                mediaSpec: MEDIA_DEFAULT_SPEC,
+                            }}
+                        >
+                            <Row>
+                                <Cell cols={13}>cols</Cell>
+                            </Row>
+                        </MediaLayoutContext.Provider>
+                    </div>,
+                );
+            };
+
+            // Act, Assert
+            expect(render).toThrow();
+
+            // eslint-disable-next-line no-console
+            console.error.mockRestore();
```

Comment:
This should be in a `afterEach()` so that it runs even if the test fails. If `mockRestore` only exists when this has been mocked, then do a check with `typeof` before calling it.